### PR TITLE
Remove reference to .theme-redesign in icons.scss

### PR DESCRIPTION
**Pull request by pdubroy**
Remove usage of `theme-redesign` class in icons.scss. Part of #20847.

### CODE REVIEW

File: client/shared/src/global-styles/icons.scss [hunk 1]
```diff
@@ -17,23 +17,16 @@
 
 // TODO find a better way to scale icons that is not proportional to the font size
 // 14px font size -> 20px icon size
```
Comment (eseliger) [inline client/shared/src/global-styles/icons.scss:19]:
```suggestion
// 14px font size -> 16px icon size
```

..if I understand this magic correctly 😬

**Review by eseliger — APPROVED** [review]

File: client/shared/src/global-styles/icons.scss (hunk 1 above)
Comment (pdubroy) [reply client/shared/src/global-styles/icons.scss]: Good catch!

**Comment by pdubroy** [comment]
Hmm, tons of changes in Percy. Must have missed something!

**Review by limitedmage — APPROVED** [review]

**Comment by pdubroy** [comment]
@5h1rU Do you have any idea what's up with the Percy changes here? I'm super confused. Especially since the icons in the "Original" snapshots appear to be 20px. It's almost as if the redesign CSS was not applied in those snapshots. Locally I'm not seeing any difference between old and new.

**Comment by 5h1rU** [comment]
> @5h1rU Do you have any idea what's up with the Percy changes here? I'm super confused. Especially since the icons in the "Original" snapshots appear to be 20px. It's almost as if the redesign CSS was not applied in those snapshots. Locally I do not see any difference between old and new.

I'm not sure, maybe `:where` pseudoselector is not allowed for snapshots, but honestly, I didn't find the problem's root. If you look at the [Percy changes for icons changes PR](https://percy.io/Sourcegraph/Sourcegraph/builds/10626140/changed/597862547?browser=chrome&browser_ids=16&subcategories=approved&viewLayout=overlay&viewMode=new&width=1920&widths=1920) Percy works as expected. But browsing in the PR history it's not getting the styles correctly.

When you back to the last snapshot [change before this PR,](https://percy.io/Sourcegraph/Sourcegraph/builds/11068465/changed/621319262?browser=chrome&browser_ids=16&subcategories=unreviewed%2Cchanges_requested&viewLayout=overlay&viewMode=new&width=1920&widths=1920) the icons have 20px.

**Comment by valerybugakov** [comment]
Closing this PR since in favor of already merged https://github.com/sourcegraph/sourcegraph/issues/24167.

**Pull request closed without merging**